staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher_whitebox_test.go

```diff
@@ -649,6 +650,7 @@ func TestCacherNoLeakWithMultipleWatchers(t *testing.T) {
 
 	// run the collision test for 3 seconds to let ~2 buckets expire
 	stopCh := make(chan struct{})
+	errCh := make(chan error, 1)
```

Comment:
nit: Can we use consistent naming throughout these tests? Is there a reason why we use `errCh` here but `errc` everywhere else below?

Reply:
Thank you for the review @kevindelgado! Defaulted to `errc` but chose `errCh` here to be consistent with `stopCh`. Overall consistency is probably preferable, adjusted name in https://github.com/kubernetes/kubernetes/pull/103023/commits/4ad641bfc761057b48be86318134c4d09655b1da.